src/main/java/uk/gov/companieshouse/bankruptofficersearch/api/controller/ScottishBankruptOfficerController.java

```diff
@@ -20,7 +21,7 @@
     private ScottishBankruptOfficerSearchServiceImpl service;
 
     @PostMapping("/internal/officer-search/scottish-bankrupt-officers")
-    public ResponseEntity<ScottishBankruptOfficerSearchResults> searchScottishBankruptOfficers(@RequestBody ScottishBankruptOfficerSearch search){
+    public ResponseEntity<ScottishBankruptOfficerSearchResults> searchScottishBankruptOfficers(@RequestBody ScottishBankruptOfficerSearch search) throws OracleQueryApiException {
```

Comment:
The controller shouldn't know Oracle query API is being interacted with. This exception should be caught in the service.

Reply:
Changed this to throw a ServiceException